comment/managers/followers.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+from comment.utils import get_username_for_comment
+
+
+class FollowerManager(models.Manager):
+    def is_following(self, email, model_object):
+        content_type = ContentType.objects.get_for_model(model_object)
+        return self.filter(email=email, object_id=model_object.id, content_type=content_type).exists()
+
+    def follow(self, email, username, model_object):
+        if not email or self.is_following(email, model_object):
+            return None
+        return self.create(email=email, username=username, content_object=model_object)
+
+    def unfollow(self, email, model_object):
+        content_type = ContentType.objects.get_for_model(model_object)
+        self.filter(email=email, object_id=model_object.id, content_type=content_type).delete()
```

Comment:
Should be using `get` method instead of `filter` when we expected only one instance here? I think it could be better performance wise.